build/generate_bundle.sh

```diff
@@ -38,6 +38,13 @@ generate_bundle() {
     echo "---- Generated bundle complete at ${WORKING_DIR}/manifests/${OPERATOR_NAME}.clusterserviceversion.yaml"
 }
 
+copy_extra_metadata() {
+    echo "-- Copy extra metadata in"
+    pushd "${REL}/../"
+    cp -r ./deploy/olm-catalog/service-telemetry-operator/tests/ "${WORKING_DIR}"
+    cp ./deploy/olm-catalog/service-telemetry-operator/metadata/properties.yaml "${WORKING_DIR}/metadata/"
+}
+
```

Comment:
We add this because our version of operator-sdk for building doesn't understand these files, but newer versions of operator-sdk (for testing purposes) does, and newer versions of `opm` (as used in both downstream and upstream index image builds) also understands these files. Just copy them into the bundle directory during building.

Reply:
Thanks for explanation. Can we please put this comment in the code instead, so that when somebody else than you will need to read the code in the future, it won't end with scratching head and lots of WTH? :).

Reply:
+1